src/mscorlib/mscorlib.shared.sources.props

```diff
@@ -753,6 +755,29 @@
     <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\Tasks\TaskToApm.cs" />
     <ThreadingSources Condition="'$(FeatureCominterop)' == 'true'" Include="$(BclSourcesRoot)\System\Threading\Tasks\IAsyncCausalityTracerStatics.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(FeatureCoreFxFileStream)' != 'true'">
+    <FileStreamSources Include="$(BclSourcesRoot)\System\IO\FileStream.cs" />
+    <SafehandleSources Include="$(BclSourcesRoot)\Microsoft\Win32\SafeHandles\SafeFileHandle.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(FeatureCoreFxFileStream)' == 'true'">
+    <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\FileStream.cs" />
+    <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\FileStream.NetStandard17.cs" />
```

Comment:
I guess it is ok for now ... but the FileStream code in CoreFX should be deleted to avoid duplication.